modular_skyrat/modules/organs/code/organs.dm

```diff
@@ -0,0 +1,8 @@
+/// Copy traits from one organ to another - e.g. with custom roundstart organs that should still get species traits applied.
+/obj/item/organ/proc/copy_traits_from(obj/item/organ/old_organ)
+	if(isnull(old_organ))
+		return
+
+	// make sure the organ gets the actions from any species specific things (like hemophage Drain Victim, etc)
+	for(var/datum/action/action as anything in old_organ.actions)
+		add_item_action(action.type)
```

Comment:
This might not be necessary, and might actually cause issues with Hemophage's tongues, once my refactor goes through.